news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2017-11-05
+--- 
+
+![](filepath) BANNER IN PROGRESS
+
+After a long hiatus, Chaos, Voli and I are happy to announce the revival of official beatmapping contests: Welcome to the _osu! Mapping Olympiad!_
+
+Let me elaborate on the concept for the osu! Mapping Olympiad. To ensure that there is a healthy and consistent flow of official contests, 
+we have worked on a concept which involved all types of modes - three contests per mode will happen annually in a specific order. 
+To keep things interesting, we have also worked on different kind of contests - so stay tuned in the upcoming months!
+
+### Songs (STILL NEED HELP WITH THE SONG PREVIEWS)
+
+To start off, we are launching this contest with not one, but **three** songs for you to tackle! Showcasing a variety of our Featured Artists, have yourself a listen to the tracks you’ll be taking on:
+
+- Kuba Oms - Beautiful Uncertainty (Redux)
+- OISHII - Honey Snowflakes
+- cySmix - Escapism
+
+There’s going to be something for everyone’s taste - funky rhythms by Kuba Oms, fast-paced goodness by OISHII and some chill beats by cYsmix!
+
+### Ruleset
+
+- This contest is for the **osu!** (standard) mode only.
+- Each entry should follow the Ranking Criteria.
+- Beatmap submissions may contain files up to 15 MB.
+- Storyboarding will not be considered in the judging.
+- You may submit **one** difficulty for **one** song. Multiple submissions for multiple songs will not be accepted.
+- You may collaborate if you like, but only the submitter of each entry will receive credit for judging and prizes.
+- Maps submitted to the contest must not be uploaded to the BSS until the contest has concluded and the winners have been announced.
+- **Send your finished beatmap as .osu-file to XXX before the deadline expired. Any submissions received after the deadline expired will not be accepted.**
+
+### Judging Criteria
+
+We have also slightly reworked the judging criteria to provide a more reasonable judging base. Down below you can see what your map will be judged for:
```

Comment:
Reworked in comparison to what? The average, not highly involved individual won't know that you're talking about past mapping contests that easily - if you dont mention them like that people won't know what this is referring to

If it's not very relevant to the point you're making then i think it's better not to mention it - a simple "All entries will be judged by the following criteria:" should be enough

Reply:
Oh lol, yeahh that's valid. I've thought too much from my own perspective